### DISCUSSION TITLE
手机号登录和账号密码登录可以用了吗？

### DISCUSSION THREAD

**Discussion by ice909** [Binaryify/NeteaseCloudMusicApi#1793] [General]
![Screenshot_2023-09-05-23-12-38-078_org.mozilla.firefox.png](https://github.com/Binaryify/NeteaseCloudMusicApi/assets/59855569/03ff5604-c641-4373-b9f0-aec15ad0a509)

我看文档里更新了

**Comment by Binaryify** [comment]
可以的

**Reply by ice909** [reply]
![image](https://github.com/Binaryify/NeteaseCloudMusicApi/assets/59855569/8459b12f-e669-4099-92c4-57f228dfc418)
我这里怎么还是需要验证啊？

**Reply by Binaryify** [reply]
自己搭的还是用的别人的,拉了代码了吗

**Reply by ice909** [reply]
自己搭的，拉了最新代码了

**Reply by Binaryify** [reply]
无法重现,估计你ip被检测了,官网登录下试试
![image](https://github.com/Binaryify/NeteaseCloudMusicApi/assets/12221718/305c212f-154e-466d-bf64-e0cb3d0ee586)

**Reply by ice909** [reply]
官网输入账号密码直接就登录了

**Reply by ice909** [reply]
> 自己搭的还是用的别人的,拉了代码了吗

我拉了代码后，使用pm2 restart 重启了一下服务，操作应该没问题吧~~~~

**Reply by Binaryify** [reply]
查一下服务器上代码的package.json的版本号,如果是新的,那就是服务器ip被检测了,我也没啥好办法,只能用二维码扫码登录了

**Reply by ice909** [reply]
完蛋，只能使用二维码登录了
